ceph-nvmeof.conf

```
@@ -17,6 +17,7 @@ state_update_notify = True
 state_update_interval_sec = 5
 #min_controller_id = 1
 #max_controller_id = 65519
+enable_discovery_controller = FALSE
```

```suggestion
enable_discovery_controller = false
```

`FALSE` does not evaluate as true, but `TRUE` would evaluate at true: [see](url) [here](https://docs.python.org/3/library/configparser.html)
```
By default when using getboolean() config parsers consider the following values 
True: '1', 'yes', 'true', 'on' and the following values False: '0', 'no', 'false', 'off'.
```